### PR TITLE
Allow custom plugin configuration

### PR DESCRIPTION
**Pull request by shanson7**
Opentsdb allows plugins access to the config file for configuration

### CODE REVIEW

**Comment by shanson7** [comment]
I can't view the travis config or logs for the failure. Current master is failing too.

**Comment by shanson7** [comment]
I checked rubocop and there were some failures, so I fixed/quieted them. That might be what travis is complaining about.

File: libraries/opentsdb_instance.rb [hunk 1]
```diff
@@ -246,7 +247,7 @@ def action_enable
               group new_resource.group
               cookbook new_resource.cookbook
               notifies :restart, new_resource, :delayed
-              mode 0644
```
Comment (acaiafa) [inline libraries/opentsdb_instance.rb]: this change is not necessary

**Comment by acaiafa** [comment]
seems reasonable outside of the minor permissions change.

File: libraries/opentsdb_instance.rb (hunk 1 above)
Comment (shanson7) [inline libraries/opentsdb_instance.rb]: Rubocop didn't like the leading 0 and wanted `0o644`. I wasn't sure if that was supported in chef, but I know that strings are.

**Comment by shanson7** [comment]
Put mode back to numeral.

**Comment by acaiafa** [comment]
+1

**Pull request merged**